pycaching/cache.py

```diff
@@ -59,6 +61,7 @@ class Cache(object):
         "hike_short": "Short hike (less than 1km)",
         "hiking": "Significant hike",
         "horses": "Horses",
+        "hqsolutionchecker": "Geocaching.com solution checker",  # id = 72
```

Comment:
I am not sure about this, as it differs from the original description `HQ solution checker`. As far as I have seen, the existing attributes are using the official text from the corresponding SVG sprite (see `pycaching.util.get_possible_attributes` method), so we should probably do this here as well.

Reply:
I used the official text from the geocaching website, but I see what you mean about the SVG sprite.

```
  {
    "id": 72,
    "name": "Geocaching.com solution checker",
    "hasYesOption": true,
    "hasNoOption": false,
    "yesIconUrl": "https://www.geocaching.com/images/attributes/hqsolutionchecker-yes.png",
    "noIconUrl": "https://www.geocaching.com/images/attributes/hqsolutionchecker-no.png",
    "notChosenIconUrl": "https://www.geocaching.com/images/attributes/hqsolutionchecker.png"
  }
```